### ISSUE TITLE
MNNConvert filed to convert TF modle to MNN

### ISSUE THREAD

**Issue by valwang** [alibaba/MNN#95]
When I tried to convert one TF model to MNN, I got following errors:

$ ./MNNConvert -f TF --modelFile note_model.pb --MNNModel note_model.mnn --bizCode MNN

MNNConverter Version: 0.0.0.1 - MNN @ 2018

Start to Convert Other Model Format To MNN Model...
terminate called after throwing an instance of 'Error'
  what():  [20:29:01] /home/valwang/project/MNN-master/tools/converter/source/tensorflow/FusedBatchNormTf.cpp:46: Check failed: channels > 0 ==> Batchnorm Channels Paramter is Wrong! 0 name: batch_normalization_1/FusedBatchNorm_1
Aborted (core dumped)

What's the reason for failure? And how to solve it? Thanks!

**Comment by czy2014hust** [comment]
check issue #63
